recipe/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
     - cartopy
     - cf-units
     - cftime
+    - compilers
```

Comment:
```suggestion
```

Reply:
not needed anymore :+1:

Reply:
This was removed in https://github.com/ESMValGroup/ESMValCore/issues/2419, but it didn't get cherry-picked onto the release branch! 😭 Should I go and cherry-pick this PR, then create a rc3?

Reply:
Removed in 448a951 😊

Reply:
nah just remove it here, no need for yet another rc3 - cheers!

Reply:
Super, thank you! Does that mean this can be merged now? 👼

Reply:
one sec - how's the pin on esmpy look like? in rc1 it was `- esmpy !=8.1.0,<8.6.0` - maybe not pin it extremo >8.6 but remove the <8.6 oe let it a bit more free

Reply:
The pin is the same as rc1 in [this repository](https://github.com/conda-forge/esmvalcore-feedstock/blob/4161a9e4e23e98e5e7aac16fc2626bcbb8f81abe/recipe/meta.yaml#L35) and in [ESMValCore](https://github.com/ESMValGroup/ESMValCore/blob/a51da9224158cfb3fec4b9c022694b30b687eb2c/environment.yml#L16) and what is currently in the [release branch](https://github.com/ESMValGroup/ESMValCore/blob/0768af8df07f530c7acbb0d3b2334c1cad6fe3ed/environment.yml#L16).

Reply:
OK prob best to leave it that way - that means that the latest `iris-esmf-regrid` will have *probable* issues as per https://github.com/ESMValGroup/ESMValCore/pull/2445 but I guess that's one way to find out, since iris-esmf devs never told us that their latest version works with older ESMPy's - I am most prob gonna build a build: 1 soon to get us rid of Python 3.9, so nay bother